org/qmckl_sherman_morrison_woodbury.org

```diff
@@ -466,20 +563,20 @@ assert(rc == QMCKL_SUCCESS);
 #include <math.h>
 #include "qmckl.h"
 
-qmckl_exit_code qmckl_woodbury_3_c(const qmckl_context context, 
-                                const uint64_t Dim,
+qmckl_exit_code qmckl_woodbury_3_c_(const qmckl_context context, 
+                                const uint64_t* Dim_p,
                                 const double* Updates,
                                 const uint64_t* Updates_index,
-                                const double breakdown,
+                                const double* breakdown_p,
                                 double * Slater_inv) {
 /*
```

Comment:
Sorry, it would be nice to use a standard notation. Either use `const double* breakdonw_p` or `const double * breakdown_p` or maybe change `double * Slater_inv` to `double* Slater_inv`, that way if we want to preprocess these using a script, it would be easier. It also looks fancy to keep things consistent.

Reply:
Yes I agree. I will remove the space between 'double' and '*'.